### DISCUSSION TITLE
how can i accept or reject an Octernship

### DISCUSSION THREAD

**Discussion by anugcodes** [community/community#49931] [GitHub Education]
i got  a mail from the github-classroom bot inviting to collaborate to an octernship with the link to the private repository. i want to know what to do next and how can i accept or reject the octernship. It also said that the invitation will expire within 7 days.

**Comment by Ntshangase** [comment]
Hello,

  you find this helpful: https://github.com/community/community/discussions/49769#discussion-4951406
